packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1253,6 +1253,31 @@ export default mixins(
 							this.nodeSelectedByName(node.name);
 						});
 					});
+
+					const tagsEnabled = this.$store.getters['settings/areTagsEnabled'];
+					if (Array.isArray(workflowData.tags) && tagsEnabled) {
+						const allTags: ITag[] = await this.$store.dispatch('tags/fetchAll');
+						const tagNames = new Set(allTags.map((tag) => tag.name));
```

Comment:
Should this not be a getter? (Just asking, never sure when.)

Reply:
just names does not seem like the most useful getter. also this does not need to be reactive... So in this case it's not needed.